src/packingSlip/controller.js

```diff
@@ -198,6 +200,21 @@ async function GetPopulatedPackingSlips(
       }
     }
 
+    if (getPending) {
+      pipeline.push({
+        $match: {
+          $or: [
+            {
+              shipment: { $eq: null },
```

Comment:
Should we check for undefined as well?

Reply:
As far as I am aware, this check should suffice in MongoDB wortld.